src/server/routes/__tests__/activities.unit.spec.js

```diff
@@ -78,7 +81,10 @@ describe('activities.unit', () => {
 
       await postActivityHandler(getActivityRequest(), getMockResponseToolkit())
 
-      expect(logger.error).toHaveBeenCalledWith('Error create activity:', error)
+      expect(logger.error).toHaveBeenCalledWith(
+        'Error creating activity',
+        error
+      )
```

Comment:
You're testing a function you've now made external to the file, which I don't think is quite right. You should mock handleServerError and handleNotFound, asserting that they're called with the expected arguments and that you get the expected response from the handler. This should be simpler than where you've been testing for the logger to have been called and then for the result to be in the expected shape.